plugin/inlay_hint.py

```diff
@@ -126,7 +127,10 @@ def format_inlay_hint_label(inlay_hint: InlayHint, session: Session, phantom_uui
                     'phantom_uuid': phantom_uuid
                 }
             })
-            result += '<a href="{command}">'.format(command=inlay_hint_click_command)
+            result += '<a href="{command}" title="Double Click {tooltip}">'.format(
```

Comment:
Actually we are adding tooltip on the outer element already if you look at the code below. So this inner one won't show (or maybe it will override the other one)? I think you should consolidate that so that we do everything in the outer tooltip.

Also, do you think "Double Click {original tooltip}" is universal enough solution? You don't know what the original tooltip will say so I feel this can be a weird Frankenstein like "Double Click Parameter Name" or something weird otherwise.

What about "{original tooltip} [double click to insert]", for example?

Reply:
>  I think you should consolidate that so that we do everything in the outer tooltip.

I didn't see this comment, but I did the opposite. I think you should consolidate that so that we do everything in the outer tooltip.
I removed the outer tooltip and moved it inside. So currently the tooltip will not override anything.

I will continue experimenting tomorrow.
If you have any suggestions, feel free to push directly on this branch.

Reply:
There are still two tooltips. One on the inner `a` and one on the outer `span`.
Also, is "Double click" clear enough? Maybe "Double-click to execute"?

Reply:
Isn't the word "execute" a bit confusing? The `InlayHint.textEdits` to insert the text from the inlay hint into the document, not to execute arbritrary commands or edits.

```typescript
	/**
	 * Optional text edits that are performed when accepting this inlay hint.
	 *
	 * *Note* that edits are expected to change the document so that the inlay
	 * hint (or its nearest variant) is now part of the document and the inlay
	 * hint itself is now obsolete.
	 *
	 * Depending on the client capability `inlayHint.resolveSupport` clients
	 * might resolve this property late using the resolve request.
	 */
	textEdits?: TextEdit[];
```

VSCode has "Double click to insert".